File: src/main/java/nl/mpcjanssen/simpletask/ThemedActivity.java
package nl.mpcjanssen.simpletask;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public abstract class ThemedActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        TodoApplication app = (TodoApplication) getApplication();
        setTheme(app.getActiveTheme());
        setTheme(app.getActiveFont());
        super.onCreate(savedInstanceState);
    }
}

File: src/main/java/nl/mpcjanssen/simpletask/ThemedActivity.kt
package nl.mpcjanssen.simpletask

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import nl.mpcjanssen.simpletask.util.Config

abstract class ThemedActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(Config.activeTheme)

        super.onCreate(savedInstanceState)
    }
}
